one.ablaze.floorp.yml

```diff
@@ -0,0 +1,39 @@
+app-id: one.ablaze.floorp
+name: Floorp
+runtime: org.freedesktop.Platform
+runtime-version: 21.08
+sdk: org.freedesktop.Sdk
+
+command: floorp
+
+finish-args:
+  - --share=ipc
+  - --socket=pcsc  
+  - --socket=x11
+  - --socket=wayland
+  - --share=network
+  - --socket=pulseaudio
+  - --filesystem=home
+  - --device=all
+  - --talk-name=org.freedesktop.FileManager1
+  - --talk-name=org.a11y.Bus 
+  - --talk-name=org.gnome.SessionManager 
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.freedesktop.ScreenSaver 
+  - --system-talk-name=org.freedesktop.NetworkManager
+  
+modules:                
+  - name: firefox
+    buildsystem: simple
+    build-commands:
+      - cd /app/ && tar Jxvf ${FLATPAK_BUILDER_BUILDDIR}/floorp.tar.xz
+      
+    sources:
+      - type: file
+        url: 'https://sda1.net/storage/floorp/tarball/8.7.2/floorp.tar.xz'
+        sha256: ee9a2c31385c56ea43cd699d1ed9569147cdd56695e567b7a374fa4f2639b1c4
+        
+      - type: file
+        path: org.kde.kst.appdata.xml
```

Comment:
that doesn't install it FWIW

Reply:
also it's the wrong name, obviously.

Reply:
Oops.
I'll fix it.

Reply:
Fixed in https://github.com/flathub/flathub/pull/3103/commits/e25ee120c3a88daa28e0d48413d351516c7f30c9